akka-http2-support/src/test/scala/akka/http/impl/engine/http2/Http2FrameProbe.scala

```diff
@@ -173,6 +175,11 @@ private[http] object Http2FrameProbe extends Matchers {
         (lastStreamId, ErrorCode.byId(reader.readIntBE()))
       }
 
+      override def expectSETTINGS(): Unit = {
+        // (6.5) The stream identifier for a SETTINGS frame MUST be zero (0x0)
+        expectFramePayload(FrameType.SETTINGS, Flags.NO_FLAGS, 0) // ignore the payload for the time being.
```

Comment:
Just return the payload as a ByteString if it is not yet parsed.

Reply:
Isn't the payload of a SETTINGS frame empty?

Reply:
I guess I'll edit the comment, then 😅

Reply:
```suggestion
        // ignore the payload for the time being, SETTINGS frame 
        // should have empty payloads anyway
        expectFramePayload(FrameType.SETTINGS, Flags.NO_FLAGS, 0)         
```